test/system/060-mount.bats

```diff
@@ -148,6 +148,14 @@ load helpers
     run_podman run --rm --mount "${mountopts},readwrite=true" $iname \
                touch /image-mount/readwrite
 
+    tmpctr="c-$(safename)"
+    subpathopts="type=image,src=$iname,dst=/image-mount,subpath=/etc"
+    run_podman run --name $tmpctr --mount "${subpathopts},readwrite=true" $iname \
+               ls /image-mount/shadow
+    run_podman inspect $tmpctr --format '{{ index .Mounts 0 }}'
+    assert "$output" =~ ".*/etc" "SubPath contains /etc"
```

Comment:
you should be able to use something like `{{ (index .Mounts 0).SubPath }}` to only list the new field then you can make an exact string match and not accept /etc anywhere in the mount path

Reply:
I had no idea you could do this.